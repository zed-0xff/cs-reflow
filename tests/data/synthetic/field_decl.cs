public class C1 {
    uint foo;

    void run() {
        if( foo >= 0 )
            always_called();
    }
}

