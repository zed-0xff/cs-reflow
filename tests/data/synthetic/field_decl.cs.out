void run()
{
    always_called();
}
